src/WorkflowDefinition.php

```diff
@@ -80,6 +82,7 @@ public function addWorkflow(AbstractWorkflow $workflow, array $dependencies = []
         $this->graph->connectGraph($definition->graph, $workflowId, $dependencies);
 
         foreach ($definition->jobs as $jobId => $job) {
+            $job['job'] = $job['job']->withJobId($workflowId . '.' . $jobId);
```

Comment:
Regarding my comment, it's this change that I can't quite place with the rest of the PR. Is this just an existing bug in Venture that doesn't have anything to do with the changes in this PR?